dialogue-serde/src/main/java/com/palantir/conjure/java/dialogue/serde/ErrorDecoder.java

```diff
@@ -56,6 +62,15 @@ boolean isError(Response response) {
     }
 
     RuntimeException decode(Response response) {
+        if (log.isDebugEnabled()) {
+            log.debug("Received an error response", diagnosticArgs(response).toArray(new Object[0]));
```

Comment:
Why the double logging on debug?

Reply:
This gives us a single place where we can enable logging instead of checking all possible places that can catch an exception.